src/fides/api/service/saas_request/override_implementations/marigold_engage_request_overrides.py

```diff
@@ -67,10 +67,12 @@ def marigold_engage_user_read(
                 method=HTTPMethod.GET,
                 path="/user",
                 query_params=signed_payload(secrets, payload),
-            )
+            ),
+            ignore_errors=[400],
```

Comment:
Because we're using a `request_override` for the user read, we can't define `ignore_errors` on the SaaS config, it needs to be defined on the `client.send` inside the request override. It's possible for a request override function to make multiple `client.send` calls so if we define `ignore_errors` at the SaaS config level, we couldn't programmatically determine which `client.send` to apply that to.

Reply:
We could define a variable that makes explicit for future programmers that we are ignoring the 400 error because the user might not exist, and also remove a magic number/list, by defining something like 

` error400ForUserNotFound = [400]`

Reply:
~~Also, Do we have an explanation from their docs on why an UserNotFound is a `400` error instead of a `404` ? Usually  NotFound errors are `404`~~. 
Nevermind, [found the reference on the Docs](https://getstarted.meetmarigold.com/engagebysailthru/Content/developers/api/user.html#ErrorCodes).   I'ts odd that they override the usual 404 error. 
Maybe we should check for the specific error message "User not found with {KEY}", just so we dont blank accept any 400 error

Reply:
Its better now. But i still think we should rename the variable, just so we have more clarity on what type of "400" error are we ignoring